logging/config/efk-logging-curator.adoc

```diff
@@ -8,7 +8,18 @@ toc::[]
 The Elasticsearch Curator tool performs scheduled maintenance operations on a per-project basis. Curator performs actions daily based on its configuration. Only one Curator Pod is
 recommended per Elasticsearch cluster.
 
-The cluster logging installation creates a Curator configuration map, `curator` in the `openshift-logging` project, which incorporates the Curator configuration file, *_curator5.yaml_* and an {product-title} custom configuration file, *_config.yaml_*.
+The Cluster Logging Operator installation creates a *Curator* ConfigMap in the `openshift-logging` project, which contains three sections: the link:https://www.elastic.co/guide/en/elasticsearch/client/curator/5.2/actionfile.html[Curator *action* file], a `config.yaml` file, and a `curator5` file.  
+
+* *actions*. In {product-title} cluster logging, the Curator `actions` file supports only the *delete_indices* action.
```

Comment:
the Curator `actions` file supports only the *delete_indices* action.
---------
I think it support all actions in  https://www.elastic.co/guide/en/elasticsearch/client/curator/current/actions.html. But only delete_indices are used and tested by openshfit.
@josefkarasek  Could you confirm?

Reply:
@anpingli @josefkarasek I made some updates to the section that Anping commented on (thanks Anping!!). 
```
* *actions.yaml*. The Curator `actions` file.
* *curator5.yaml*. The Curator configuration file for general configuration, such as the Elasticsearch hostname, client connection details, and the path to certificates.
* *config.yaml*. A legacy {product-title} configuration file for Curator. Supports only the Curator *delete_indices* action

```